modules/core/client/app/init.js

```diff
@@ -1,80 +1,43 @@
-'use strict';
+(function (app) {
```

Comment:
send in document and window and remove app since it is now window.applicationConfiguartion